docs/Protos Storage.md

```diff
@@ -81,6 +81,19 @@ pub struct Proto<TAccountId, TBlockNumber> {
 	pub accounts_info: AccountsInfo,
 }
 ```
+### Metadata keys
+Supported metadata keys are 'image', 'title', 'json_attributes', 'description’. 
```

Comment:
We need to document each of those.
What do you expect as format in your front-end? Text, image? For what are they used for? etc...